### PR TITLE
[core] Add more infos to each breakpoint

### LINKED ISSUE #48129: [Core][Debugger] Info about node IP for each breakpoint is not exposed in Ray Debugger
### Description

When using the Ray Debugger in multi-node settings where we hit numerous breakpoints, it would be helpful to see which node/IP matches which breakpoint.

### Use case

_No response_

### PR DESCRIPTION
**Pull request by rynewang**
Ray debugger CLI (`ray debug`) is useful but when you have multiple tasks in the breakpoint, and they are of the same task function, one can have difficulties understanding which is which.

Add  "Node ID", "Worker ID", "Actor ID", "Task ID" to the breakpoint info so in `ray debug` these IDs show up in the breakpoint selection menu. This helps users disambiguate between active breakpoints.

Closes #48129.

### CODE REVIEW

File: python/ray/scripts/scripts.py [hunk 1]
```diff
@@ -246,7 +252,18 @@ def debug(address):
...
+        table = [
+            [
+                "index",
+                "timestamp",
+                "Ray task",
+                "filename:lineno",
+                "Node ID",
+                "Worker ID",
+                "Actor ID",
+                "Task ID",
```
Comment (ruisearch42) [inline python/ray/scripts/scripts.py:264]: These should be ordered by usefulness. Should `Task ID` come earlier and `Node ID` come last?

**Review by ruisearch42 — APPROVED** [review]
LGTM. Thanks for the change.

Question: should we make this configurable at some point? Sometimes people may not need all these and it could be too verbose?

**Comment by rynewang** [comment]
verbose mode added.

**Review by ruisearch42 — APPROVED** [review]
nice

**Pull request merged**